### ISSUE TITLE
error creating retail client in 1.6.0

### ISSUE THREAD

**Issue by manfer** [irontec/ivozprovider#239]
I had done a fresh install of bleeding and it doen't allow to create retail clients. I get an "error saving record" message

Fresh debian jessie install

```
cd /etc/apt/sources.list.d
echo deb http://packages.irontec.com/debian bleeding main extra > ivozprovider.list
echo deb http://packages.irontec.com/debian chloe main > klear.list
wget http://packages.irontec.com/public.key -q -O - | apt-key add -
apt-get update
apt-get install ivozprovider
```

When are the repositories updated with new code?

I don't know if there is some problem because the repositories are not fully updated yet or if there is any bug.

Is there any log I can read to try to figure out what is happening? Nothing I can see inspecting the browser console and network, nothing on apache logs.

**Comment by Kaian** [comment]
Hi @manfer 

Changes on bleeding are packaged every commit. After merging the PR of the changes they are ready in a couple of minutes.

But **just now** (really, just now) I'm merging bleeding changes in oasis branch, and changing versions in bleeding to match the new release in roadmap (1.6.0). I suggest you to wait a couple of minutes so the release process has finished (it will change from pre-release to release [here](https://github.com/irontec/ivozprovider/releases))

Sorry for the disturbance!

**Comment by Kaian** [comment]
Hi,

I have updated all packages in the bleeding repository and tested, and you were right!
Outgoing DDI rules implemented in #196 was present in Retail Clients screen while they shouldn't. I have corrected the problem and backported to stable branch.

Please, run apt-get update and apt-get upgrade and ensure all the package version of ivozprovider are 1.6.0 with `dpkg -l | grep ivozprovider`

Isos will be ready with that change in a couple of minutes ;-)

Thanks for the early testing!!

**Comment by manfer** [comment]
Solved
